### PR TITLE
Add Junit

### PR DESCRIPTION
**Pull request by bnjm2000**
Add Junit tests for:
Parser
DeleteParser
ExitCommand
HelpCommand

Fixes part of #14

### CODE REVIEW

File: src/test/java/seedu/parser/ParserTest.java [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outputStream));
+
+        parser.process(validResponse);
+
+        System.setOut(System.out);
+        String output = outputStream.toString();
+
+        String expectedOutput = "no commands found. Enter \"help\" for a list of commands." + System.lineSeparator() +
+                "____________________________________________________________" + System.lineSeparator();
```
Comment (yingx9) [inline src/test/java/seedu/parser/ParserTest.java]: Your output has no line divider " "____________________________________________________________"  but your expecting a line divider.
image

Comment (bnjm2000) [reply src/test/java/seedu/parser/ParserTest.java]: I have already made that change...
Screenshot 2023-10-16 at 4 16 54 PM

File: src/test/java/seedu/parser/ParserTest.java [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+    public void testProcessHelpCommand() {
+        Parser parser = new Parser();
+        String validResponse = "help";
+
+        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outputStream));
+        parser.process(validResponse);
+
+        System.setOut(System.out);
+        String output = outputStream.toString();
```
Comment (yingx9) [inline src/test/java/seedu/parser/ParserTest.java:44]: Do you want to extract this to a new class like TestUtils then put this code as a method which returns the output, so all the test methods can use this to get the output message. If not this code repeats a lot

Comment (yingx9) [reply src/test/java/seedu/parser/ParserTest.java:44]: Also I want to use this part for my testing too 👀

File: src/main/java/seedu/parser/Parser.java [hunk 3]
```diff
@@ -41,7 +41,8 @@ public void process(String response) {
                 System.out.println(e.getMessage());
             }
         } else {
-            System.out.println("no commands found. Enter \"help\" for a list of commands.\n");
+            System.out.println("no commands found. Enter \"help\" for a list of commands.\n"+
```
Comment (yingx9) [inline src/main/java/seedu/parser/Parser.java:44]: change the \n to System.lineSeparator()

**Comment by yingx9** [comment]
All looks good now!

**Pull request merged**